diff --git a/backend/drm/meson.build b/backend/drm/meson.build
index 5d2f2b1f..0d74e412 100644
--- a/backend/drm/meson.build
+++ b/backend/drm/meson.build
@@ -27,7 +27,7 @@ hwdata_dir = hwdata.get_variable(pkgconfig: 'pkgdatadir')
 pnpids_c = custom_target(
 	'pnpids.c',
 	output: 'pnpids.c',
-	input: files(hwdata_dir / 'pnp.ids'),
+	input: '/app/usr/share/hwdata/pnp.ids',
 	feed: true,
 	capture: true,
 	command: files('gen_pnpids.sh'),
